packages/cli-repl/src/format-output.spec.ts

```diff
@@ -172,7 +178,7 @@ for (const colors of [ false, true ]) {
           type: 'ShowCollectionsResult'
         }));
 
-        expect(output).to.contain('nested_documents\ndecimal128\ncoll\npeople_imported   [view]\ncats              [time-series]');
+        expect(output).to.equal('nested_documents\ndecimal128\ncoll\npeople_imported   [view]\ncats              [time-series]');
```

Comment:
:+1:

Reply:
accident. meant to change the ones I was changing. But if it works I suppose there's no harm ;)